backend/challenges/RenameColumnAlternative/challenge.py

```diff
@@ -0,0 +1,27 @@
+import hypothesis.strategies as st
+from hypothesis.extra.pandas import data_frames, column, range_indexes
+import numpy as np
+import pandas as pd
+import pandas.testing as tm
+
+import Challenge from ??
+
+
+def transform(df: pd.DataFrame) -> pd.DataFrame:
+    df = df.rename(columns={"Value": "NewValue"})
+    return df
+
+# Since the "run" "initial" and "test" use cases all rely on the same param generators it makes sense to share them
+# Possibly as a dict 
+params = {
+    'df': data_frames(
+        columns=[column("Value", dtype=np.dtype(int))],
+        rows=st.tuples(st.sampled_from([1, 2, 3, 4, 5, 6, 7, 8, 9]),),
+        index=range_indexes(min_size=3, max_size=10),
+    )
+}
+
+# The "run" and "initial" methods are always the same regardless of challenge
```

Comment:
Since we would need to send the generic `Challenge` class to the user, a `BaseClass` (as we already had) would be the better solution for this because it enforces the interface